.github/workflows/roster-integration.yml

```diff
@@ -0,0 +1,61 @@
+name: Roster Integration Workflow
+
+on:
+  pull_request:
+    branches:
+      - master
+
+env:
+  SELENIUM_GRID_PROVIDER: saucelabs
+  CLOUD_WATCH_METRIC: false
+  TEST_TYPE: Github-Action
+  SAUCE_USERNAME: ${{secrets.SAUCE_USERNAME}}
+  SAUCE_ACCESS_KEY: ${{secrets.SAUCE_ACCESS_KEY}}
+
+jobs:
+  integ-roster:
+    name: Roster Integration Test
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout Package
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Pack the React Components Library and install the tarball into the Test Demo
+        run: |
+          current_version=$(.github/script/get-current-version)
+          echo "Packing current version:" $current_version
+          npm ci
+          npm run build
+          npm pack
+          cd integration/app/test-demo
+          npm uninstall amazon-chime-sdk-component-library-react
+          npm install ../../../amazon-chime-sdk-component-library-react-$current_version.tgz
+      - name: Create a Job ID
+        id: create-job-id
+        uses: filipstefansson/uuid-action@ce29ebbb0981ac2448c2e406e848bfaa30ddf04c
+      - name: Set JOB_ID Env Variable
+        run: echo "JOB_ID=${{ steps.create-job-id.outputs.uuid }}" >> $GITHUB_ENV
+      - name: Echo Job ID
+        run: echo "${{ steps.create-job-id.outputs.uuid }}"
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: us-east-1
+      - name: Setup Sauce Connect
+        uses: saucelabs/sauce-connect-action@v1
+        with:
+          username: ${{ secrets.SAUCE_USERNAME }}
+          accessKey: ${{ secrets.SAUCE_ACCESS_KEY }}
+          noSSLBumpDomains: all
+          tunnelIdentifier: ${{ steps.create-job-id.outputs.uuid }}
+      - name: Clean Install
+        run: npm ci
+      - name: Run Roster Integration Test
```

Comment:
nit: Can we say the browser name here `Run Roster Integration Test`

Reply:
Sure, so the browser used in this GitHub Action is Chrome, should we change it to `Run Roster Integration Test - Chrome`? Sounds good?

Reply:
Updated